iceoryx_posh/include/iceoryx_posh/internal/roudi/introspection/port_introspection.inl

```diff
@@ -199,38 +197,34 @@ bool PortIntrospection<SenderPort, ReceiverPort>::PortData::updateConnectionStat
 
 template <typename SenderPort, typename ReceiverPort>
 bool PortIntrospection<SenderPort, ReceiverPort>::PortData::addSender(typename SenderPort::MemberType_t* port,
-                                                                      const std::string& name,
+                                                                      const ProcessName_t& name,
                                                                       const capro::ServiceDescription& service,
-                                                                      const std::string& runnable)
+                                                                      const RunnableName_t& runnable)
 {
-    std::string serviceId = static_cast<cxx::Serialization>(service).toString();
-
     std::lock_guard<std::mutex> lock(m_mutex);
 
-    auto iter = m_senderMap.find(serviceId);
+    auto iter = m_senderMap.find(service);
     if (iter != m_senderMap.end())
```

Comment:
I know you didnt modify it but I just 'saw `{}` is missing here. I can do that in #360 so we get this PR merged and can merge PR #379 as well?!
Multiple times in this file, of course!